solaris/solaris11/postinstall.sh

```diff
@@ -1,16 +1,17 @@
 #!/bin/sh
 # postinst script for wazuh-agent
-# Wazuh, Inc 2015-2020
+# Wazuh, Inc 2015-2021
 
 install_path="<INSTALL_PATH>"
 osversion=$(uname -v)
 SCA_BASE_DIR="${install_path}/ruleset/sca/"
 SCA_TMP_DIR="${install_path}/tmp/sca"
 
+# upgrade from 3
 if [ -d ${install_path}/logs/ossec ]; then
-  if [ -z "$(ls -A ${install_path}/logs/ossec)" ]; then
+  if [ -z "$(ls -A ${install_path}/logs/ossec)" ]; then  # empty
```

Comment:
Do you consider necessary the comment? `-z` is known that returns true if the value is empty.

Reply:
I do not remember adding these comments, but it seems that I put them at some point. Removed in acc587f0aa60ee65e0d73900caede7e7237055aa